Makefile

```diff
@@ -51,11 +51,33 @@ fix-all:
 	make black-fix
 	make lint-fix
 
+# Download binaries from Taskcluster to run tests
+# See https://firefox-ci-tc.services.mozilla.com/tasks/index/translations.cache.level-1.toolchains.v3.fast-align/latest
+download-bin:
+	BIN=bin bash utils/download-bin.sh
+
 # Run unit tests
+# Some tests work only on Linux, use Docker if running locally on other OS
 run-tests:
 	poetry install --only tests --only utils --no-root
 	PYTHONPATH=$$(pwd) poetry run pytest tests -vv
 
+# Run unit tests locally under Docker
+run-tests-docker:
+  	# build docker images for apple silicon
+	if [ $$(uname -m) == 'arm64' ]; then \
+	  	export DOCKER_DEFAULT_PLATFORM=linux/amd64; \
+	fi
+
+	docker build \
+		--file taskcluster/docker/base/Dockerfile \
+		--tag ftt-base .
+	docker build \
+		--build-arg DOCKER_IMAGE_PARENT=ftt-base \
+		--file taskcluster/docker/test/Dockerfile \
+		--tag ftt-test .
+	docker run -it -v $$(pwd):/builds/worker/checkouts -w /builds/worker/checkouts  ftt-test make run-tests
```

Comment:
This is failing on my macOS Ventura 13.4.1:

```
➤ make run-tests-docker
if [ $(uname -m) == 'arm64' ]; then \
	  	export DOCKER_DEFAULT_PLATFORM=linux/amd64; \
	fi
docker build \
		--file taskcluster/docker/base/Dockerfile \
		--tag ftt-base .
[+] Building 53.1s (10/10) FINISHED                                                                                                                                                                                                                                                                                                                                                                                docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                                                                                                                                                               0.0s
 => => transferring dockerfile: 1.75kB                                                                                                                                                                                                                                                                                                                                                                                             0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                                                                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                                                                                                                                                                    0.0s
 => [internal] load metadata for docker.io/library/ubuntu:22.04                                                                                                                                                                                                                                                                                                                                                                    1.1s
 => [1/6] FROM docker.io/library/ubuntu:22.04@sha256:f9d633ff6640178c2d0525017174a688e2c1aef28f0a0130b26bd5554491f0da                                                                                                                                                                                                                                                                                                              1.8s
 => => resolve docker.io/library/ubuntu:22.04@sha256:f9d633ff6640178c2d0525017174a688e2c1aef28f0a0130b26bd5554491f0da                                                                                                                                                                                                                                                                                                              0.0s
 => => sha256:f9d633ff6640178c2d0525017174a688e2c1aef28f0a0130b26bd5554491f0da 1.13kB / 1.13kB                                                                                                                                                                                                                                                                                                                                     0.0s
 => => sha256:ca165754e2f953a4f686409b1eb5855212f42a252462c9c50bbc3077f3b9a654 424B / 424B                                                                                                                                                                                                                                                                                                                                         0.0s
 => => sha256:a50ab9f167975489853cbffd2be3bcadab3a9da27faf390ac48603c60d5c59e7 2.31kB / 2.31kB                                                                                                                                                                                                                                                                                                                                     0.0s
 => => sha256:a4a2c7a57ed8b997579870d0927433b03cfd94f5dba2153bdbcd885b5d620035 27.36MB / 27.36MB                                                                                                                                                                                                                                                                                                                                   1.0s
 => => extracting sha256:a4a2c7a57ed8b997579870d0927433b03cfd94f5dba2153bdbcd885b5d620035                                                                                                                                                                                                                                                                                                                                          0.6s
 => [2/6] RUN mkdir /builds &&     useradd -d /builds/worker -s /bin/bash -m worker &&     chown worker:worker /builds/worker &&     mkdir /builds/worker/artifacts &&     chown worker:worker /builds/worker/artifacts                                                                                                                                                                                                            0.2s
 => [3/6] WORKDIR /builds/worker/                                                                                                                                                                                                                                                                                                                                                                                                  0.0s
 => [4/6] RUN apt-get update -qq     && apt-get install -y tzdata     && apt-get install -y python3                           python3-pip                           python3-yaml                           locales                           git                           tmux                           htop                           vim                           nano                           zstd                        46.8s
 => [5/6] RUN locale-gen "en_US.UTF-8"                                                                                                                                                                                                                                                                                                                                                                                             1.0s
 => [6/6] RUN pip install zstandard                                                                                                                                                                                                                                                                                                                                                                                                1.0s
 => exporting to image                                                                                                                                                                                                                                                                                                                                                                                                             1.1s
 => => exporting layers                                                                                                                                                                                                                                                                                                                                                                                                            1.1s
 => => writing image sha256:9a06d6281301ae64f54bed0ab5aa977eef17008fb1c6b9ae39125c75e4cb5ca0                                                                                                                                                                                                                                                                                                                                       0.0s
 => => naming to docker.io/library/ftt-base                                                                                                                                                                                                                                                                                                                                                                                        0.0s

What's Next?
  View summary of image vulnerabilities and recommendations → docker scout quickview
docker build \
		--build-arg DOCKER_IMAGE_PARENT=ftt-base \
		--file taskcluster/docker/test/Dockerfile \
		--tag ftt-test .
[+] Building 12.6s (7/7) FINISHED                                                                                                                                                                                                                                                                                                                                                                                  docker:desktop-linux
 => [internal] load .dockerignore                                                                                                                                                                                                                                                                                                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                                                                                                                                                                    0.0s
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                                                                                                                                                               0.0s
 => => transferring dockerfile: 477B                                                                                                                                                                                                                                                                                                                                                                                               0.0s
 => [internal] load metadata for docker.io/library/ftt-base:latest                                                                                                                                                                                                                                                                                                                                                                 0.0s
 => [1/4] FROM docker.io/library/ftt-base                                                                                                                                                                                                                                                                                                                                                                                          0.0s
 => [2/4] RUN apt-get update -qq     && apt-get install -y wget     && apt-get install -y pigz     && apt-get install -y unzip     && apt-get install -y zstd     && apt-get clean                                                                                                                                                                                                                                                 6.4s
 => [3/4] RUN pip install poetry                                                                                                                                                                                                                                                                                                                                                                                                   6.0s
 => ERROR [4/4] RUN chown worker:worker /builds/worker/.cache                                                                                                                                                                                                                                                                                                                                                                      0.2s
------
 > [4/4] RUN chown worker:worker /builds/worker/.cache:
0.190 chown: cannot access '/builds/worker/.cache': No such file or directory
------
Dockerfile:16
--------------------
  14 |     VOLUME /builds/worker/checkouts
  15 |     VOLUME /builds/worker/.cache
  16 | >>> RUN chown worker:worker /builds/worker/.cache
  17 |
--------------------
ERROR: failed to solve: process "/bin/sh -c chown worker:worker /builds/worker/.cache" did not complete successfully: exit code: 1
make: *** [run-tests-docker] Error 1
```

Reply:
It should work now